packaging/leapp-repository.spec

```diff
@@ -4,15 +4,12 @@
 %global sos_report_plugindir %{python2_sitelib}/sos/plugins/
 %global py3_sos_report_plugindir /usr/lib/python3.6/site-packages/sos/plugins/
 # Defining py_byte_compile macro because it is not defined in old rpm (el7)
-# Only defined to python2 since python3 is not used in RHEL7
-%{!?py_byte_compile: %global py_byte_compile py2_byte_compile() {\
-    python_binary="%1"\
-    bytecode_compilation_path="%2"\
-    find $bytecode_compilation_path -type f -a -name "*.py" -print0 | xargs -0 $python_binary -c 'import py_compile, sys; [py_compile.compile(f, dfile=f.partition("$RPM_BUILD_ROOT")[2]) for f in sys.argv[1:]]' || :\
+%{!?py_byte_compile: %global py_byte_compile() {\
+    python_binary="%1";\
+    bytecode_compilation_path="%2";\
+    find $bytecode_compilation_path -type f -a -name "*.py" -print0 | xargs -0 $python_binary -c 'import py_compile, sys; [py_compile.compile(f, dfile=f.partition("$RPM_BUILD_ROOT")[2]) for f in sys.argv[1:]]' || :;\
```

Comment:
Not that `"$RPM_BUILD_ROOT"` is within `'` quotes, so shell will not expand it. This was later fixed in Fedora, yet not in the retired package you took it from, but here: https://src.fedoraproject.org/rpms/python-rpm-macros/blob/master/f/macros.pybytecompile